client/state/sites/selectors.js

```diff
@@ -71,6 +71,22 @@ export const getSite = createSelector(
 			domain: getSiteDomain( state, siteId ),
 			is_previewable: isSitePreviewable( state, siteId )
 		};
+
+		if ( attributes.jetpack ) {
```

Comment:
I'm sure we could. My question to you is where would that live? We call out to `lib/site/computer-attributes.js` to get the main site attributes. But, all of the Jetpack attributes rely on state and selectors that in the `state/sites/selectors.js` file.

If I created a `lib/site/jetpack-computer-properties.js` helper, that would still need to import several selectors from the state file, and I would need to pass state to this helper file.

One other option I thought of is adding the helper directly in the selectors file, but every function there is exported, and we wouldn't want to export this helper.

I'm glad to take a pass at this, I'm just not sure where to start.